content/tutorials/custom-zk-chain/20.customizing-your-chain.md

```diff
@@ -0,0 +1,393 @@
+---
+title: Customizing your ZK chain
+description: Create a ZK chain with a custom ERC20 base token.
+
+---
+
+One way you can customize your ZK chain is by changing the base token used for gas from ETH to an ERC20 token.
+Let's try customizing a new chain by using a custom ERC20 base token.
+
+With the initial release of ZK stack, custom ERC20 base tokens must be added to an allowlist for use as a base token for a chain.
+The allowed addresses are stored and checked in the `BridgeHub` contract on the L1.
+In the future it will be possible to add a new token in a permissionless process.
+
+For now, you have the ability to add any tokens to the allowlist in your local ecosystem.
```

Comment:
it sort of expands on it - the sentence above says there is a limitation that will be changed in the future, and this says you can still try it out locally (since you run the ecosystem)